res/index.html

```diff
@@ -7,7 +7,7 @@
 <meta charset="utf-8">
 <meta name="viewport" content="initial-scale=1" />
 <title>perf.html</title>
-<script src='/analytics.js'></script>
+<script src='/js/analytics.js'></script>
```

Comment:
As said below as well, this script is (for now) still at the root of the `dist` directory so its URL shouldn't be changed here.

(I admit the devtools' network panel is quite confusing here, as we don't see the request -- but the chrome's devtools show it clearly)